### PR TITLE
Reordering and simplifying the README

### PR DESCRIPTION
**Pull request by lebca**
- more consistent headings
- de-duplicating mf install instructions
- reordering Get Started higher up, easier to find
- de-duplicating documentation links
- lowering Resources section, easier to find

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,52 +18,28 @@ As a developer, you can also use MetricFlow's interfaces to construct APIs for i
...
-mf health-checks # confirm your datawarehouse connection is working
-```
+To get up and running with your own metrics, you should rely on MetricFlow’s documentation available at [MetricFlow docs](https://docs.transform.co/docs/metricflow/guides/introduction).
 
-Run the tutorial:
+### Tutorial
 
 ```
-mf tutorial # optionally add `--skip-ds` if you have already confirmed your datawarehouse connection works
+> mf tutorial # optionally add `--skip-ds` if you have already confirmed your datawarehouse connection works
```
Comment (belindabennett) [inline README.md:42]: `--skip-dw`

File: README.md [hunk 2]
```diff
@@ -115,6 +91,14 @@ You might wish to read our [code of conduct](http://community.transform.co/metri
 
 To get started on direct contributions, head on over to our [contributor guide](CONTRIBUTING.md)
```
Comment (belindabennett) [inline README.md:92]: add period to the end

**Comment by belindabennett** [comment]
Thank you for this! It looks great! Added two comments: 
- 1 fix
- 1 nit

**Comment by belindabennett** [comment]
Also: looks like you need to merge in the postgres info!

**Review by tlento — APPROVED** [review]
One nit inline.

I think a separate PR for the community README issue is in order, but I'm happy to re-review an additional commit here.

**Pull request merged**